sync/client/client_test.go

```diff
@@ -30,6 +31,16 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
+var networkCodec codec.Manager
+
+func init() {
+	var err error
+	networkCodec, err = message.NewCodec(message.BlockSyncSummary{})
```

Comment:
nit: should we name this testCodec?
we could also have a function like newTestCodec(t), but I don't know if it is worth this change here given the other code follows the pattern with `init`

Reply:
hmm it could be a bit weird to export it for tests but not for prod code.

Reply:
I intentionally avoided to export it so that it won't be mistakenly imported from message pkg everytime.